src/Components/NavBar/index.tsx

```diff
@@ -0,0 +1,386 @@
+              </Sans>
+              <Spacer py={0.5} />
+              <Separator />
+            </Box>
+          )}
+
+          <Box py={title ? 0 : 1}>{children}</Box>
+        </Flex>
+      </BorderBox>
+    </MenuContainer>
+  )
+}
+
+const NavMenu = () => {
+  return (
+    <Menu title="More">
+      <MenuItem>Artists</MenuItem>
+      <MenuItem>Shows</MenuItem>
+      <MenuItem>Museums</MenuItem>
+      <MenuItem>Artsy for Galleries</MenuItem>
+    </Menu>
+  )
+}
+
+const NotificationMenu = () => {
+  return (
+    <Menu title="Actvity">
+      <MenuItem>
+        <Flex alignItems="center">
+          <Box width={40} height={40} bg="black5" mr={2} />
+          <Box>
+            <Sans size="2">5 works added</Sans>
+            <Sans size="2" weight="medium">
+              Jeff Koons
+            </Sans>
+          </Box>
+        </Flex>
+      </MenuItem>
+      <MenuItem>
+        <Flex alignItems="center">
+          <Box width={40} height={40} bg="black5" mr={2} />
+          <Box>
+            <Sans size="2">5 works added</Sans>
+            <Sans size="2" weight="medium">
+              Josef Albers
+            </Sans>
+          </Box>
+        </Flex>
+      </MenuItem>
+      <MenuItem>
+        <Flex alignItems="center">
+          <Box width={40} height={40} bg="black5" mr={2} />
+          <Box>
+            <Sans size="2">5 works added</Sans>
+            <Sans size="2" weight="medium">
+              Ellsworth Kelly
+            </Sans>
+          </Box>
+        </Flex>
+      </MenuItem>
+      <Flex p={2} flexDirection="column" alignItems="center">
+        <Separator />
+        <Box pt={2}>
+          <Sans size="2">
+            <Link href="/" style={{ textDecoration: "underline" }}>
+              View all
+            </Link>
+          </Sans>
+        </Box>
+      </Flex>
+    </Menu>
+  )
+}
+
+const UserMenu = () => {
+  return (
+    <Menu>
+      <MenuItem>
+        <HeartIcon top={5} mr={0.5} /> Saves & Follows
+      </MenuItem>
+      <MenuItem>
+        <SoloIcon top={4} mr={0.5} /> Collector Profile
+      </MenuItem>
+      <MenuItem>
+        <SettingsIcon top={5} mr={0.5} /> Settings
+      </MenuItem>
+      <MenuItem>
+        <PowerIcon top={5} mr={0.5} /> Log out
+      </MenuItem>
+    </Menu>
+  )
+}
+
+const NavItem: React.SFC<NavItemProps> = ({
+  display = "block",
+  children,
+  href,
+  menu,
+  onClick,
+}) => {
+  return (
+    <Box
+      p={1}
+      display={display}
+      position="relative"
+      onClick={onClick}
+      style={{ cursor: "pointer" }}
+    >
+      <Sans size="3" weight="medium">
+        {href ? <Link href="/">{children}</Link> : children}
+      </Sans>
+
+      {menu}
+    </Box>
+  )
+}
+
+enum navActions {
+  TOGGLE_NAV_MENU,
+  TOGGLE_USER_MENU,
+  TOGGLE_NOTIFICATIONS_MENU,
+  TOGGLE_MOBILE_MENU,
+  TOGGLE_LOG_IN,
+}
+
+const navReducer = (state, action) => {
+  const resetMenus = {
+    showNavMenu: false,
+    showeUserMenu: false,
```

Comment:
Electronic User? ;)

Reply:
> _Of engelond to caunterbury they wende,
The hooly blisful martir for to seke,
That hem hath holpen whan that they were seeke.
Bifil that in that seson on a day..._

Practicing a bit of my old english :)

Reply:
🇬🇧